ra2ce/graph/networks.py

```diff
@@ -80,12 +81,25 @@ def __init__(self, config, files):
             self.region_var = None
 
         # Cleanup
-        self.snapping = config["cleanup"]["snapping_threshold"]
-        self.segmentation_length = config["cleanup"]["segmentation_length"]
+        _cleanup_options = config.get("cleanup", {})
+        if any(_cleanup_options.items()) and any(
+            [v for k, v in _cleanup_options.items()]
+        ):
+            _cleanup_options["cleanup"] = True
```

Comment:
I don't really see the need for this assignment, either way it should be inside the `_setup_cleanup_options`

Reply:
This shall be cleaned up in the `NetworkShpWrapper`